server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/build_detail.scss

```diff
@@ -290,22 +290,63 @@ div.build_detail_container_content {
 .widget iframe {
     border: none;
 }
+
+#console-action-bar {
+    min-height: 20px;
+    padding: 10px;
+
+    background-color: #333;
+
+    div {
+        float: right;
+    }
+
+    a {
+        padding-right: 15px;
+        color: yellow;
+    }
+
+    &.white-theme {
+        background-color: white;
+        a {
+            color: #1F66BD;
```

Comment:
Color literals like `#1F66BD` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#1F66BD` should be written as `#1f66bd`